core/models.py

```diff
@@ -139,7 +138,7 @@ class Meta:
 class Notifiable(UserManager):
     async def anotify(self):
         async for user in self.iterator():
-            send_new_hackathon_email.delay(user)
```

Comment:
Any reason for this change?

Reply:
I made the change because the parameter is no longer user, and I don't know what I should be setting it to.
`def send_hackathon_emails(frequency):`
where `frequency` is a string that is either "weekly" or "monthly"

Reply:
ah, okay okay. Sure we can fix that later. Just make a comment about that in the code.